SQF/dayz_code/stringtable.xml

```diff
@@ -9772,6 +9772,11140 @@
 			<English>M4A1 FL</English>
 			<Russian>M4A1 Ф</Russian>
 		</Key>
```

Comment:
@Streatman - You accidentally duplicated a ton of strings that were already in the stringtable.